ssl/quic/quic_port.c

```diff
@@ -1290,8 +1308,8 @@ static int port_validate_token(QUIC_PKT_HDR *hdr, QUIC_PORT *port,
         goto err;
     time_diff = ossl_time2seconds(ossl_time_abs_difference(token.timestamp,
```

Comment:
I know this is existing code but the type of `time_diff` is wrong. It should be `uint64_t`